packages/atomic/src/components/common/pager/pager-common.tsx

```diff
@@ -0,0 +1,134 @@
+import {EventEmitter, h} from '@stencil/core';
+import ArrowRight from '../../../images/arrow-right.svg';
+import {FocusTargetController} from '../../../utils/accessibility-utils';
+import {randomID} from '../../../utils/utils';
+import {Button} from '../button';
+import {AnyBindings} from '../interface/bindings';
+import {RadioButton} from '../radio-button';
+import {Pager, SearchStatus} from '../types';
+
+interface SearchStatusOptions {
+  bindings: AnyBindings;
+  initializeSearchStatus(): SearchStatus;
+  initializePager(): Pager;
+  getEventEmitter(): EventEmitter | undefined;
+  getActivePage(): FocusTargetController | undefined;
+}
+
+export class PagerCommon {
```

Comment:
Did you mean to include this in your PR?

Reply:
no, I don't know what I've done with these branches, complete mess!!